### ISSUE TITLE
Problems with shoes and heels

### ISSUE THREAD

**Issue by SuNNLov** [itsamewayu/OLFallenDoll#1119]
**Describe the bug**
There are two things I'm finding strange...

Shoes and heels cannot be removed or permanently disabled in all experiments. Even if you remove your shoes or heels in one experiment, they will always be active again when you start another experiment that supports them.

Additionally, when removing heels in free mode, the character still acts as if she were wearing them, with invisible heels supporting the characters.

And all of the characters' shoes and heels act strangely, are clipping the feet and strangely move instead of staying static.

This can also be seen in advanced suits outfits or outfits that fully cover the body, like Elizabeth's advanced suit, when equipped, the character's feet do not behave like heels.

**Desktop (please complete the following information):**
 - OS: Win 11 Pro
 - Game Version: 0.7.9

**Comment by yuanyuankuang** [comment]
question 1, not a BUG,the setting is like this.
question 2, I can't review it ,may be  resolved,may be you can show me a video .

**Comment by SuNNLov** [comment]
> question 1, not a BUG,the setting is like this. question 2, I can't review it ,may be resolved,may be you can show me a video .

Regarding question 2, I made an example video showing the strange behavior similar to what I had been seeing, including this happening in Elizabeth and Celia's advanced costumes...

https://github.com/itsamewayu/OLFallenDoll/assets/130314204/51bc0a03-05df-491c-8521-fb9776ac1252

![image](https://github.com/itsamewayu/OLFallenDoll/assets/130314204/a037eaae-47eb-4748-aed0-6c3a888c7994)![image](https://github.com/itsamewayu/OLFallenDoll/assets/130314204/f001dd4b-9c93-4da4-88ba-7f2e5fbe4269)

**Comment by yuanyuankuang** [comment]
YES,I see.Because the shoes is not a part of the clothing，SO ,shoes disable ,check it next version.
